src/shared/codelens/localLambdaRunner.ts

```diff
@@ -420,7 +420,8 @@ export async function invokeLambdaFunction(
         templatePath: invokeArgs.samTemplatePath,
         eventPath,
         environmentVariablePath,
-        invoker: samLocalInvokeCommand
+        invoker: samLocalInvokeCommand,
+        dockerNetwork: config.dockerNetwork
```

Comment:
A similar change will be necessary around line 185 in the same file. (The other code path is used by javascript SAM Applications)

Reply:
Ah, whoops - missed that in my going through the files.  Thanks, I'll update the PR.